newIDE/app/src/InstancesEditor/index.js

```diff
@@ -604,20 +609,40 @@ export default class InstancesEditor extends Component<Props> {
     this.pixiRenderer.view.focus();
   };
 
+  _onRightClickedOnInstance = (coordinates: {|
+    offsetX: number,
+    offsetY: number,
+    x: number,
+    y: number,
+  |}) => {
+    this._onRightClicked({
+      ...coordinates,
+      ignoreSelectedObjectNamesForContextMenu: false,
+    });
+  };
+
   _onRightClicked = ({
     offsetX,
     offsetY,
     x,
     y,
+    ignoreSelectedObjectNamesForContextMenu,
   }: {|
     offsetX: number,
     offsetY: number,
     x: number,
     y: number,
+    ignoreSelectedObjectNamesForContextMenu?: boolean,
   |}) => {
     this.lastContextMenuX = offsetX;
     this.lastContextMenuY = offsetY;
-    if (this.props.onContextMenu) this.props.onContextMenu(x, y);
+    if (this.props.onContextMenu) {
+      const _ignoreSelectedObjectNamesForContextMenu =
+        typeof ignoreSelectedObjectNamesForContextMenu === 'boolean'
+          ? ignoreSelectedObjectNamesForContextMenu
+          : false;
+      this.props.onContextMenu(x, y, _ignoreSelectedObjectNamesForContextMenu);
```

Comment:
```suggestion
      this.props.onContextMenu(x, y, ignoreSelectedObjectNamesForContextMenu || false);
```
or 
```suggestion
      this.props.onContextMenu(x, y, !!ignoreSelectedObjectNamesForContextMenu);
```

Should work (undefined == falsy)